doc/modules/model_evaluation.rst

```diff
@@ -676,10 +676,14 @@ score is equal to the classification accuracy.
   >>> from sklearn.metrics import jaccard_similarity_score
   >>> y_pred = [0, 2, 1, 3]
   >>> y_true = [0, 1, 2, 3]
-  >>> jaccard_similarity_score(y_true, y_pred)
```

Comment:
There is a comment above about being equal to accuracy that needs to be removed. Rather you need to document that this function works like prfs as a naively set-wise measure applying only to binary/multilabel targets, for which multiclass problems are binarized. Need to mention averages available akin to prfs